### ISSUE TITLE
Affected_extensions returning nil

### ISSUE THREAD

**Issue by danmanstx** [vassilevsky/face_control#2]
Currently trying with the latest version which is `0.5.0` and i am getting the following error 

```
/Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/lib/face_control/inputs/rubocop_json.rb:20:in `block (2 levels) in comments': undefined method `include?' for nil:NilClass (NoMethodError)
    from /Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/lib/face_control/inputs/rubocop_json.rb:19:in `reject'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/lib/face_control/inputs/rubocop_json.rb:19:in `block in comments'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/lib/face_control/inputs/rubocop_json.rb:18:in `map'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/lib/face_control/inputs/rubocop_json.rb:18:in `comments'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/exe/face-control:30:in `block in <top (required)>'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/exe/face-control:29:in `each'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/gems/face_control-0.5.0/exe/face-control:29:in `<top (required)>'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/bin/face-control:23:in `load'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/bin/face-control:23:in `<main>'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/bin/ruby_executable_hooks:15:in `eval'
    from /Users/administrator/.rvm/gems/ruby-2.2.2/bin/ruby_executable_hooks:15:in `<main>'
```

it looks like the `affected_extensions` method is not being called correctly as i can't call it from irb using a proper pull_request object, and is returning nil when being called by the gem.

**Comment by vassilevsky** [comment]
Wait, are you telling me that someone uses this besides myself?! Wow!

I'm so sorry I missed this report. I will look into it ASAP.

**Comment by vassilevsky** [comment]
I have released version 0.5.1 with a fix for this problem and then version 0.6.0 with additions to documentation and some small refactoring. I apologise for this silly bug. Please update the gem and let me know if it works.

I also force-pushed the correct master branch to this repo. Previously it had unreleased changes that confused you. Please see the actual README now.

**Comment by danmanstx** [comment]
cool thanks! i will try today!

**Comment by vassilevsky** [comment]
I released version 0.7.0 with new architecture. You no longer need to run the checkers manually. `face-control` runs them, and only with affected files.
